# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/mysqlcc/qmakespec.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2003-12-13 20:10:43.000000000 -0300
+++ ./configure.in	2005-02-12 11:37:12.733286108 -0300
@@ -25,7 +25,7 @@
 AC_SUBST(MYSQLCC_SOURCES)
 AC_OUTPUT(mysqlcc.pro)
 if test -z "${QMAKESPEC}"; then
-  QMAKESPEC=$QTDIR/mkspecs/$host_os
+  QMAKESPEC=$QTDIR/mkspecs/linux-g++
 else
   QMAKESPEC=${QMAKESPEC}
 fi
